src/main/java/com/mercadolibre/grupo1/projetointegrador/exceptions/ExceptionCatchIsEmpty.java

```diff
@@ -0,0 +1,10 @@
+package com.mercadolibre.grupo1.projetointegrador.exceptions;
+/*
+@author Gabriel Essenio
+Cria exceçao caso lista esteja vazia
+ */
+public class ExceptionCatchIsEmpty extends RuntimeException {
```

Comment:
Vamo padronizar as exceptions para DESCRICAO + Exception
```suggestion
public class ListIsEmptyException extends RuntimeException {
```

Reply:
Beleza!
Vou mudar a descrição para a padronização